elizaOS/eliza
# docs: Add Tagalog README Translation

# Relates to:

#1419 

# Risks

Low - This is a documentation-only change adding a new translation file.

# Background

Adds Tagalog translation of the main README.md file as README_TG.md, including:
- Full translation of all main sections
- Proper Tagalog character rendering and formatting
- Maintained all links and functionality from the original README

## What kind of change is this?

Improvements (adds new language support to existing documentation)

# Documentation changes needed?

My changes are documentation changes themselves - adding a new translated README file.

# Testing

## Where should a reviewer start?

- Open README_TG.md in either VSCode/Cursor/github.dev's Markdown preview
- Verify formatting is correct
- Check that all links are functional
- Verify Tagalog text renders properly

## Detailed testing steps

- View the raw file to verify proper Tagalog text formatting
- View the rendered file to ensure readability and consistency
- Check that the language selector at the top includes a Tagalog link
- Verify all internal links work correctly
- Ensure images and formatting match the main README.md structure
